.travis.yml

```diff
@@ -113,9 +113,10 @@ script:
     - export PYTHONPATH=/tmp/$TRAVIS_JOB_ID/lib/python$TRAVIS_PYTHON_VERSION/site-packages
     # install easybuild-framework to unique temporary location using prepared sdist tarball
     - mkdir -p $PYTHONPATH; easy_install --prefix /tmp/$TRAVIS_JOB_ID $TRAVIS_BUILD_DIR/dist/easybuild-framework*tar.gz
-    # actively break "import setuptools", since EasyBuild should not have a runtime requirement on setuptools
-    - echo 'raise NotImplementedError("setuptools is not available")' > /tmp/$TRAVIS_JOB_ID/setuptools.py
-    - export PYTHONPATH=/tmp/$TRAVIS_JOB_ID:$PYTHONPATH
+    # make sure there are no (top-level) "import setuptools" or "import pkg_resources" statements,
+    # since EasyBuild should not have a runtime requirement on setuptools
+    - SETUPTOOLS_IMPORTS=$(egrep -RI '^(from|import)[ ]*pkg_resources|^(from|import)[ ]*setuptools' easybuild/ || true)
```

Comment:
Without the tab you don't need [] around the space.

Reply:
Strictly speaking, no, but it's awkward not to do so imho...

Reply:
Ok, will leave it so then.